jerry-main/main-unix.c

```diff
@@ -342,6 +341,8 @@ static cli_opt_t main_opts[] =
                .help = "export literals found in parsed JS input (in C source format)"),
   CLI_OPT_DEF (.id = OPT_EXEC_SNAP, .longopt = "--exec-snapshot", .argc = 1, .meta = "FILE", .quant = CLI_QUANT_A,
                .help = "execute input snapshot file(s)"),
+  CLI_OPT_DEF (.id = OPT_EXEC_SNAP_FUNC, .longopt = "--exec-snapshot-func", .argc = 2, .meta = "FILE idx",
```

Comment:
I'd suggest all metas to be all caps ("FILE IDX") so that they stand out better in the usage string